package.json

```diff
@@ -6,6 +6,7 @@
     "app-config",
     "app-config-webpack-plugin",
     "app-config-inject",
+    "react-native-app-config-transformer",
```

Comment:
`react-native-*-transformer` seems to be the convention so far, though `app-config-react-native-transformer` would look nicer in the mono repo.

Reply:
Yeah, I like `app-config-*` as a folder name better (the package can still be convention)

Reply:
I like it 👍. Updated.